src/easyscore.ts

```diff
@@ -162,7 +161,7 @@ export class Grammar {
     return { token: "['][^']*[']" };
   }
   NOTENAME(): Rule {
-    return { token: '[a-gA-G]' };
+    return { token: '[a-gA-GrR]' };
```

Comment:
Does note name = R result in a auto aligned rest? Good idea! EasyScore should definitely be as easy as possible.

Reply:
There is in the code a strange condition to align rests:

https://github.com/0xfe/vexflow/blob/3eea242fc47c7d540b226e969b6ad6a984ee5f7e/src/formatter.ts#L347

The position must be B/4 or R/4. I prefered to put support to R in easyscore. It is also quite strange that they mus be in the octave 4. 

In order to get it working, you also have to formatOptions like in the example below:

```js
const system: System = vf.System({ x: x, y: y, width: measure.width, formatOptions: { align_rests: true } });
```

your idea is very good, R => auto align rest. @0xfe what do you think?

Reply:
I'm okay with the distinction, but I'm not sure overloading `NOTENAME` is the right approach. There's `TYPE | TYPES` below which does rests.

Reply:
Also, can we separate this change from the documentation?

Reply:
Done